html/index.html

```diff
@@ -395,52 +395,60 @@ <h2>About <br> Myself</h2>
         
         
             </ul>
+
+            <div class="section-background">
+
+                <div class="section">
         
-            <div class="section">
-        
-                <div class="contact-me-section">
-        
-                    <p id="contact-me">Contact me</p>
-        
-                    <p id="contact-msg"> 
-                        If you have an application you are 
-                        interested in developing, a feature 
-                        that you need built or a project that
-                        needs coding. I’d love to help with it 
-                    </p>
-        
-                </div>
-        
-        
-                <div class="form-section">
-        
-                    <form action="">
-        
-                        <input type="text" placeholder="Toyyib" class="text"> <br>
-                        <input type="email" placeholder="Simpleshaikh2@gmail.com" class="text"> <br>
-                        <textarea type="text" placeholder="write your message heres" id="type-msg"> </textarea><br>
-        
-                    </form>
-        
-                    <div class="btn-sec">
-        
-                        <button class="sec-btn">   
-                            Get in touch
-                        </button>
+                    <div class="contact-me-section">
+            
+                        <p id="contact-me">Contact me</p>
+            
+                        <p id="contact-msg"> 
+                            If you have an application you are 
+                            interested in developing, a feature 
+                            that you need built or a project that
+                            needs coding. I’d love to help with it 
+                        </p>
             
-                        <span class="dot"></span>
-        
                     </div>
+            
+            
+                    <div class="form-section">
+            
+                        <form action="https://formspree.io/f/mayzpyer" method="post">
+            
+                            <input type="text" name="name" placeholder="Name" class="text" maxlength="30" required> <br>
+                            <input type="email" placeholder="Email" class="text"  required> <br>
+                            <textarea type="text" placeholder="write your message here" cols="30" rows="10" maxlength="500" id="type-msg" required> </textarea><br>
+                            
```

Comment:
- Lastly, please refer to the Figma design and ensure that the space between the input elements are similar to that of the Figma design :+1: 
- [ OPTIONAL ] From the images below you can see that your input elements look smaller in size compared to that of the Figma design and the `textarea` looks bigger in your design as compared to the Figma design. I will suggest that you adjust these so they look similar to that of the Figma design :100:  

| Your Design | Figma Design |
|--------|--------|
| ![Screenshot from 2023-04-04 15-01-15](https://user-images.githubusercontent.com/102699958/229834701-0bff669b-fe44-4219-89b1-804b341d8b9f.png) | ![Screenshot from 2023-04-04 14-57-57](https://user-images.githubusercontent.com/102699958/229834334-9a9aafa2-aba5-4e0a-b6c8-3e5e118d3c4b.png) |